server/routes/update-goal/models/UpdateGoalPostModel.ts

```diff
@@ -15,6 +15,10 @@ export class StepModel {
 }
 
 export default class UpdateGoalPostModel {
+  @Expose()
```

Comment:
Do we need Expose() here? Shouldn't it be exposed by default?

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/199/commits/427fbb28b69b11c9d45de2c96503d8008ebaeb50